.github/workflows/helm-test.yml

```diff
@@ -124,7 +124,8 @@ jobs:
     - name: List changed charts
       id: list-changed
       run: |
-        changed=$(ct list-changed --config "${CT_CONFIGFILE}")
+        latestRelease=$(git describe --abbrev=0 --tags)
+        changed=$(ct list-changed --config "${CT_CONFIGFILE}" --since "${latestRelease}")
```

Comment:
This same workflow runs on both `main` and PRs, right? So does this mean that this case is always true for every commit made on every branch since the last release... i.e. it can never be false except for the release commit itself?

Reply:
It actually checks if the chart itself has changed. So a change to an example or a change to the top-level README.md wont trigger a test.